### ISSUE TITLE
[question] Is there a way to stop and switch application bettwen the "pisignage" and other program

### ISSUE THREAD

**Issue by nicolasdonbosco** [colloqi/piSignage#171]
I have install the pisignage on top of the raspberry pi os (using player2/method 2)
I want to switch between my app such "chrominum" and pisignage (automatically switch to pisignage while device idle/no user interaction)
is there a way to do that?

and can I temporarily stop the pisignage? I have tried using the "pkill" command, and the piSignage automaticaly re-start again (refer to downgrade.sh script in the "shell-script" folders)
pkill -f forever
pkill -f node
pkill -f uzbl
pkill -f omx

**Comment by colloqi** [comment]
You could do pm2 stop piSignage and pm2 start piSignage

**Comment by hitesh83** [comment]
> 

as you mentioned chromium , you can just schedule your URL in pisigange itself , so no need to switch between apps.
for any other app as mentioned above pm2 needs to be used.

**Comment by nicolasdonbosco** [comment]
thanks, pm2 stop/start is worked, but can I use original UI from raspberry OS when the app is stopped? (now when I stop the pisignage, the screen just show black screen, and some option when right click on mouse pressed) 
![image](https://user-images.githubusercontent.com/78315384/162378173-dc042b70-a640-4fb1-a911-f400ffa6b152.png)

**Comment by bontorhumala** [comment]
hi @nicolasdonbosco , did you mean after you run
`pm2 stop piSignage`

you cannot see Chromium browser / Desktop at all? I also have same problem...

**Comment by nicolasdonbosco** [comment]
> hi @nicolasdonbosco , did you mean after you run `pm2 stop piSignage`
> 
> you cannot see Chromium browser / Desktop at all? I also have same problem...

yes, I can't se the chromium / desktop at all, here's the details
![image](https://user-images.githubusercontent.com/78315384/162410116-1bbee120-aab6-4426-9e8f-cdc871744c2d.png)

**Comment by colloqi** [comment]
You could press F2 or ctrl+alt+T to get the terminal and issue "chromium-browser" command
